File: 3- Second Year (2022-2023)/Term 1/OOP1 (CS311)/Lab/Labs/Lab 2/Task 1/src/Task1.java
/*Write a Java program that takes three numbers as input then calculates and prints the 
average of the numbers and display the max and min number */

import java.util.Scanner;
public class Task1 {
    public static void main(String[] args) throws Exception {
        Scanner input = new Scanner(System.in);
        System.out.println("Enter 3 numbers as inputs");
        double n1 = input.nextDouble();
        double n2 = input.nextDouble();
        double n3 = input.nextDouble();
        System.out.println("The average is " + (n1 + n2 + n3)/3.0);
        input.close();
    }
}

File: 2- First Year (2021-2022)/Term 2/Fundementals of Programming (CS221)/Lab/Lab 1/Task 1/Task 1.cpp
#include <iostream>
using namespace std;

int main()
{
	int size, sum=0, avg=0, count=0;
	cout << "Enter your array size: ";
	cin >> size;
	int arr[size];
	for (int i = 0; i < size; i++)
	{
		cout << "Enter item #" << i + 1 << " value: ";
		cin >> arr[i];
		count++;
	}
	
	//sum
	for (int i = 0; i < size; i++)
	{
		sum += arr[i];
	}
	
	cout << "Sum of numbers = " << sum << endl;
	//average
	for (int i = 0; i < size; i++)
	{
		avg = sum/count;
	}
	cout << "Average of numbers = " << avg << endl;
	
	//max
	int max = arr[0], min = arr[0];

	for (int i = 0; i < size; i++)
	{
		if (arr[i] > max)
		{
			max = arr[i];
		}
	}
	cout << "Max number = " << max << endl;
	
	//min
	for (int i = 0; i < size; i++)
	{
		if (arr[i] < min)
		{
			min = arr[i];
		}
	}
	cout << "Min number = " << min << endl;
		
	return 0;
}
